homework7/hw7task2/project2_/task2_backspace_compare.py

```diff
@@ -0,0 +1,43 @@
+"""
+Given two strings. Return if they are equal when both are typed into
+empty text editors. # means a backspace character.
+Note that after backspacing an empty text, the text will continue empty.
+Examples:
+    Input: s = "ab#c", t = "ad#c"
+    Output: True
+    # Both s and t become "ac".
+    Input: s = "a##c", t = "#a#c"
+    Output: True
+    Explanation: Both s and t become "c".
+    Input: a = "a#c", t = "b"
+    Output: False
+    Explanation: s becomes "c" while t becomes "b".
+"""
+
+
+def handle_with_backspaces(string: str) -> list:
+    """One backspace(#) deletes one symbol which is before him in string.
+
+    Return a list of symbols which were not deleted by backspaces.
+    """
+
+    backspace = "#"
+    symbols = list(string)
```

Comment:
You don't need to wrap iterable to list to iterate over it.  And you can pass only iterable to the `list`.

Reply:
Yes, I know. I forgot to remove list here. I used it before because I needed to delete backspace and symbol before backspace. List is mutable. But string no and every time after replacing symbols with '' (nothing) new object was created, that is why I used list. But here I really forgot to remove it, string will be enough.